EDU-18136 feat/ new fastcheckout intro doc (#700)
vtexdocs/help-center-content: docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout.md

---

> title: 'Introdução ao Buyer Portal Checkout'

Reviewer: ```suggestion
title: 'Buyer Portal Checkout'
```

status: PUBLISHED
createdAt: 2026-04-24T18:00:00.000Z
updatedAt: 2026-04-24T18:00:00.000Z
publishedAt: 2026-04-24T18:00:00.000Z

> firstPublishedAt: 2026-04-24T18:00:00.000Z

Reviewer: ```suggestion
```

contentType: tutorial
productTeam: Shopping

> slugEN: buyer-portal-checkout-introduction

Reviewer: ```suggestion
slugEN: buyer-portal-checkout
```

locale: pt
---

> ℹ️ Esta funcionalidade está disponível apenas para lojas que utilizam o [B2B Buyer Portal](https://help.vtex.com/pt/docs/tutorials/b2b-buyer-portal-pt), que atualmente está disponível para contas selecionadas.

> O **Buyer Portal Checkout** é a nova versão do checkout VTEX, projetado para oferecer uma experiência de finalização de compra rápida e customizável. A solução foi construída com base nas melhores práticas de usabilidade do mercado, e diretamente integrada aos fluxos de B2B do Buyer Portal como [Contratos](https://help.vtex.com/pt/docs/tutorials/b2b-buyer-portal-pt#contratos), [Campos contábeis](https://help.vtex.com/pt/docs/tutorials/b2b-buyer-portal-pt#campos-contabeis), [Budgets](https://help.vtex.com/pt/docs/tutorials/b2b-buyer-portal-pt#budgets) e [Buying Policies](https://help.vtex.com/pt/docs/tutorials/b2b-buyer-portal-pt#buying-policies).

Reviewer: ```suggestion
O **Buyer Portal Checkout** é a nova versão do checkout VTEX, projetado para oferecer uma experiência de finalização de compra rápida e customizável. A solução foi construída com base nas melhores práticas de usabilidade do mercado e diretamente integrada aos fluxos de B2B do Buyer Portal como [Contratos](https://help.vtex.com/pt/docs/tutorials/b2b-buyer-portal-pt#contratos), [Campos contábeis](https://help.vtex.com/pt/docs/tutorials/b2b-buyer-portal-pt#campos-contabeis), [Budgets](https://help.vtex.com/pt/docs/tutorials/b2b-buyer-portal-pt#budgets) e [Buying Policies](https://help.vtex.com/pt/docs/tutorials/b2b-buyer-portal-pt#buying-policies).
```

![Buyer Portal Checkout](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout_1.png)

## Conceito

> O Buyer Portal Checkout apresenta uma arquitetura mais moderna, desenvolvida para proporcionar maior flexibilidade, segurança e escalabilidade à experiência de compra. Entre as características que definem a solução, destacam-se:

Reviewer: ```suggestion
O Buyer Portal Checkout apresenta uma arquitetura moderna, desenvolvida para proporcionar maior flexibilidade, segurança e escalabilidade à experiência de compra. Entre as características que definem a solução, destacam-se:
```

- Integração com ferramentas compliance do Buyer Portal.

> 

Reviewer: ```suggestion
```

## Funcionalidades

O Buyer Portal Checkout entrega todas as funcionalidades esperadas de um checkout moderno, cobrindo o fluxo completo do carrinho até a confirmação do pedido. Entre as principais, destacam-se:

- Gerenciamento do carrinho com visualização e ajuste de itens (quantidade, adição, remoção individual ou em massa), agrupamento por canal de entrega (envio ou retirada em loja) e tratamento automático de itens indisponíveis.
- Resumo do pedido em tempo real, com detalhamento de subtotal, frete, descontos, promoções, impostos, vales-presente e total final.
- Simulação de envio direto no carrinho com endereço novo ou salvo.
- Suporte a múltiplos métodos de pagamento, incluindo cartão de crédito com parcelamento e cartões salvos (com fluxo B2B sem CVV), PIX com QR code e finalização com Google Pay ou one-click checkout.
- Revisão e confirmação do pedido com possibilidade de editar cada seção, status do pedido (aprovado ou pendente), número, resumos de entrega ou retirada e pagamento.
- Autenticação flexível do comprador, com login por email e senha ou código único.

> Além desses recursos, foram desenvolvidas novas funcionalidades para otimizar a experiência de compra de cenários B2B, facilitando a personalização e oferecendo segurança na transação para seus compradores.

Reviewer: ```suggestion
Além desses recursos, a VTEX oferece funcionalidades para otimizar a experiência de compra de cenários B2B, facilitando a personalização e oferecendo segurança na transação para seus compradores.
```

### Budgets

> A funcionalidade de [Budgets](https://help.vtex.com/pt/docs/tutorials/b2b-buyer-portal-pt#budgets) permite que o comprador veja quais são os orçamentos associados ao seu pedido antes do fechamento da compra. Isso permite que ele tenha visibilidade sobre seus fundos contábeis e evite consumir além do determinado pela organização.

Reviewer: evite consumir além ou seja impedido de consumir além?

![Cobrança da conta](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout_3.png)

### Checkout para Punchout

> O fluxo de [Punchout](https://developers.vtex.com/docs/guides/punchout) que compradores integrem seus sistemas de eprocurement diretamente ao site de ecommerce doa loja. O [Checkout para Punchout](https://developers.vtex.com/docs/guides/punchout-cart-integration) da VTEX é adaptado para esse tipo de integração, que permite que o usuário retorne o carrinho montado e finalize a compra pelo seu sistema de eprocurement.

Reviewer: ```suggestion
O fluxo de [Punchout](https://developers.vtex.com/docs/guides/punchout) possibilita que compradores integrem seus sistemas de eprocurement diretamente ao site do ecommerce. O [Checkout para Punchout](https://developers.vtex.com/docs/guides/punchout-cart-integration) da VTEX é adaptado para esse tipo de integração, que permite que o usuário retorne o carrinho montado e finalize a compra pelo seu sistema de eprocurement.
```

### Endereços de cobrança

A funcionalidade Endereços de cobrança permite que os compradores selecionem explicitamente o endereço de cobrança correto durante o checkout. A funcionalidade garante que o endereço de cobrança registrado no pedido reflita com precisão a realidade fiscal da transação e seja propagado de forma consistente para os sistemas posteriores.

![Endereços de cobrança](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout_3.png)

### Entrega agendada

A [Entrega agendada](/docs/pt/tutorials/configurar-entrega-agendada-fastcheckout) permite às lojas VTEX oferecer aos clientes a possibilidade de escolher o melhor dia e horário para receber suas compras. Essa configuração é realizada diretamente nas políticas de envio e definem os períodos disponíveis para entregas.

### Buying Policies

> O sistema de [Buying Policies](https://help.vtex.com/pt/docs/tutorials/buying-policies) suporta regras dinâmicas que os compradores podem configurar para inserir suas regras de compliance dentro do processo de compra. [Gestores de organizações](https://help.vtex.com/pt/docs/tutorials/b2b-buyer-portal-pt#gestao-organizacional) compradoras podem definir regras customizadas para que pedidos sejam encaminhados para aprovação antes de serem confirmados, ou mesmo sejam impedidos de serem completados no ato da compra.

Reviewer: ```suggestion
O sistema de [Buying Policies](https://help.vtex.com/pt/docs/tutorials/buying-policies) suporta regras dinâmicas que os compradores podem configurar para inserir suas regras de compliance dentro do processo de compra. [Gestores de organizações compradoras](https://help.vtex.com/pt/docs/tutorials/b2b-buyer-portal-pt#gestao-organizacional) podem definir regras customizadas para que pedidos sejam encaminhados para aprovação antes de serem confirmados, ou mesmo sejam impedidos de serem completados no ato da compra.
```

### Assembly Options

O recurso de [Assembly Options](https://help.vtex.com/pt/docs/tutorials/assembly-options) permite agrupar itens de um kit em um único elemento na lista do carrinho. Esse elemento exibe informações detalhadas sobre a composição do kit e oferece controle das quantidades de cada item.

![Assembly Options](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout_4.png)

### Cartões salvos

> Buyer Portal Checkout permite a utilização de cartões de créditos compartilhados pela organização. Ele também permite a possibilidade de [salvar cartões](https://developers.vtex.com/docs/api-reference/checkout-custom-card-payment-api) de usuários que possuam essa permissão para que possam ser utilizados em transações futuras. Integrado ao [fluxo de tokenização de cartões de crédito](https://help.vtex.com/pt/docs/tutorials/dpan-e-fpan-entendendo-a-seguranca-no-fluxo-de-pagamentos-tokenizados-online) da VTEX, as informações são processadas segundo as regras de [PCI](https://help.vtex.com/pt/docs/tutorials/o-que-e-o-pci-ssc), fornecendo segurança ao processo.

Reviewer: ```suggestion
O Buyer Portal Checkout permite a utilização de cartões de crédito compartilhados pela organização. Ele também possibilita [salvar cartões](https://developers.vtex.com/docs/api-reference/checkout-custom-card-payment-api) para que possam ser utilizados em transações futuras (disponível para usuários que possuam essa permissão).

De forma integrada ao [fluxo de tokenização de cartões de crédito](https://help.vtex.com/pt/docs/tutorials/dpan-e-fpan-entendendo-a-seguranca-no-fluxo-de-pagamentos-tokenizados-online) da VTEX, as informações são processadas segundo as regras de [PCI](https://help.vtex.com/pt/docs/tutorials/o-que-e-o-pci-ssc), fornecendo segurança ao processo.
```

> Buyer Portal Checkout permite a utilização de cartões de créditos compartilhados pela organização. Ele também permite a possibilidade de [salvar cartões](https://developers.vtex.com/docs/api-reference/checkout-custom-card-payment-api) de usuários que possuam essa permissão para que possam ser utilizados em transações futuras. Integrado ao [fluxo de tokenização de cartões de crédito](https://help.vtex.com/pt/docs/tutorials/dpan-e-fpan-entendendo-a-seguranca-no-fluxo-de-pagamentos-tokenizados-online) da VTEX, as informações são processadas segundo as regras de [PCI](https://help.vtex.com/pt/docs/tutorials/o-que-e-o-pci-ssc), fornecendo segurança ao processo.

Reviewer: talvez especificar a permissão ou linkar para guia de permissions

### Endereços de entrega salvos

> No Buyer Portal Checkout, usuários podem selecionar endereços de entrega a partir da [lista fornecida pela sua organização](https://help.vtex.com/pt/docs/tutorials/b2b-buyer-portal-pt#enderecos-e-gestao-de-pagamentos). Caso tenham as permissões, configuradas, usuários também podem registrar novos endereços e editar existentes diretamente na experiência de compra do checkout.

Reviewer: ```suggestion
No Buyer Portal Checkout, usuários podem selecionar endereços de entrega a partir da [lista fornecida pela sua organização](https://help.vtex.com/pt/docs/tutorials/b2b-buyer-portal-pt#enderecos-e-gestao-de-pagamentos). Caso tenham as permissões configuradas, usuários também podem registrar novos endereços e editar existentes diretamente na experiência de compra do checkout.
```

> No Buyer Portal Checkout, usuários podem selecionar endereços de entrega a partir da [lista fornecida pela sua organização](https://help.vtex.com/pt/docs/tutorials/b2b-buyer-portal-pt#enderecos-e-gestao-de-pagamentos). Caso tenham as permissões, configuradas, usuários também podem registrar novos endereços e editar existentes diretamente na experiência de compra do checkout.

Reviewer: em "permissões" adicionar link para doc de storefront permissions

![Gerenciador de Endereços](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout_5.png)

### Múltiplas opções de entrega (Multidelivery)

> A funcionalidade permite que os compradores escolham diferentes combinações de envio e [opções de envio](https://help.vtex.com/pt/docs/tutorials/opcoes-de-envio-beta) para cada item, proporcionando mais flexibilidade para a finalização da compra.

Reviewer: "diferentes combinações de envio (...) para cada item"? acho que ficou meio confuso. talvez mudar o final para "para os itens no carrinho"

![Multidelivery](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout_6.png)

### Campos contábeis

> [Campos de dados customizaveis](https://help.vtex.com/pt/docs/tutorials/campos-contabeis), configurados pela organização compradora para coletar informações administrativas ou financeiras, como centro de custo, número de pedido interno (PO) e outros dados de controle.

Reviewer: ```suggestion
[Campos de dados customizáveis](https://help.vtex.com/pt/docs/tutorials/campos-contabeis) podem ser configurados pela organização compradora para coletar informações administrativas ou financeiras, como centro de custo, número de pedido interno (PO) e outros dados de controle.
```

![Campos contábeis](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout_7.png)

> ### Pickup Store

Reviewer: ```suggestion
### Store Pickup
```

---
title: 'Introdução ao Buyer Portal Checkout'
status: PUBLISHED
createdAt: 2026-04-24T18:00:00.000Z
updatedAt: 2026-04-24T18:00:00.000Z
publishedAt: 2026-04-24T18:00:00.000Z
firstPublishedAt: 2026-04-24T18:00:00.000Z
contentType: tutorial
productTeam: Shopping
slugEN: buyer-portal-checkout-introduction
locale: pt
---

> ℹ️ Esta funcionalidade está disponível apenas para lojas que utilizam o [B2B Buyer Portal](https://help.vtex.com/pt/docs/tutorials/b2b-buyer-portal-pt), que atualmente está disponível para contas selecionadas.

> O **Buyer Portal Checkout** é a nova versão do checkout VTEX, projetado para oferecer uma experiência de finalização de compra rápida e customizável. A solução substitui a experiência anterior do [Checkout VTEX](/pt/docs/tutorials/checkout-vtex-visao-geral) por um fluxo baseado nas melhores práticas de usabilidade do mercado, com foco em aumentar a taxa de conversão das lojas e simplificar a personalização da jornada de compra.

Reviewer 2: Tem algum direcionamento para chamar de Buyer Portal Checkout? Pensara que só continuaríamos com o termo Checkout (removendo-se o termo "fast").
Sobre a parte `A solução substitui a experiência anterior do [Checkout VTEX]`, não diria que substitui, mas que agora agrega a experiência B2B no checkout.

Author: Foi definido nessa [thread](https://vtex.slack.com/archives/C080XN77T33/p1777054105984539) esse novo nome do FastCheckout, ainda mais que ele tá sendo exclusivo com o uso do Buyer Portal

Reviewer 3: Sobre o nome ✅
Sobre `A solução substitui a experiência anterior do [Checkout VTEX]`, concordo com o thaynan, karen, minha sugestão é algo na linha:
`A solução foi construída com base nas melhores práticas de usabilidade do mercado, e diretamente integrada aos fluxos de B2B do Buyer Portal - Contracts, Accounting Fields, Budgets e Buying Policies.`
Detalhe que para B2B, aumentar a taxa de conversão não é necessariamente o maior ganho que um checkout deve oferecer, mas sim integridade da informação e segurança da transação.

![Buyer Portal Checkout](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout_1.png)

## Conceito

O Buyer Portal Checkout apresenta uma arquitetura mais moderna, desenvolvida para proporcionar maior flexibilidade, segurança e escalabilidade à experiência de compra. Entre as características que definem a solução, destacam-se:

> - Fluxo de compra otimizado para conversão.

Reviewer 3: Melhor retirar esse ponto. Para B2B, como comentei, maiores ganhos são:
`- Integração com ferramentas compliance do Buyer Portal`

> - Alta performance, mesmo em carrinhos com muitos itens.

Reviewer 2: Tenho receio de afirmar isso! Até então não aumentamos a performance, e tbm não temos dados que comprovem o aumento de conversão.

- Otimização para dispositivos móveis e desktop.
- Suporte a opções omnichannel sofisticadas.

> > ℹ️ O Buyer Portal Checkout está em fase beta fechado e, no momento, está disponível apenas para lojas localizadas nos Estados Unidos. Para conhecer as condições de adesão, entre em contato com o [Suporte VTEX](https://help.vtex.com/pt/support).

Reviewer 2: Diria que no momento está disponível apenas para lojas da jornada B2B. Ademais, não estamos mais nos limitando a EUA, dado que estamos apresentando e oferecendo pra clientes do BRA, MEX e CAN.

Reviewer 3: RT com o thaynan. Mais preciso é:
`O Buyer Portal Checkout está em fase de beta fechado e, no momento, está disponível para clientes B2B na América do Norte. Para conhecer as condições de adesão, entre em contato com...`

## Funcionalidades

> Conheça as principais funcionalidades do Buyer Portal Checkout, que foram desenvolvidas para otimizar a experiência de compra, facilitar a personalização e aumentar a conversão da sua loja.

Reviewer 3: Ponto aqui novamente sobre "aumentar a conversão da sua loja". Alternativa:
`oferecer segurança de transação para seus compradors`

### Informações de contato

> A funcionalidade Informações de contato permite que o comprador especifique as informações do destinatário na experiência de entrega. A funcionalidade busca atender a diferentes cenários, como entrega única, múltiplas entregas — em que diferentes produtos são entregues por métodos distintos — e casos de uso B2C ou B2B.

Reviewer 3: Acho que podemos tirar esse ponto aqui, não é tão relevante assim como os listados abaixo..

![Informações de contato](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout_2.png)

### Cobrança da conta

> A funcionalidade de Cobrança da Conta permite que o comprador escolha um método de pagamento que gera uma cobrança diretamente para a conta da empresa. O pagamento dessa cobrança é processado por meio do fluxo de [punchout](#punchout).

Reviewer 2: Fiquei na duvida aqui. Na verdade, o punchout não seria a integração com um fornecedor?

Reviewer 2: Se estou entendendo bem, o "Cobrança de Conta" seria o budget?

Reviewer 3: Também não entendi essa funcionalidade... fiquei na dúvida entre promissória, budgets ou punchout. Independente, acho que pode reframe para que seja Budgets mesmo. Sugestão:

> Budgets

> A funcionalidade de Budgets permite que o comprador veja quais são os orçamentos associados ao seu pedido antes do fechamento da compra. Isso permite que ele tenha visibilidade sobre seus fundos contábeis e evite consumir além do determinado pela organização.

![Cobrança da conta](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout_3.png)

> ### Punchout

Reviewer 3: > Checkout para Punchout
Para ser mais preciso aqui

> O fluxo de [Punchout do carrinho](https://developers.vtex.com/docs/guides/punchout-cart-integration) permite que empresas integrem seus sistemas de eprocurement diretamente ao site de e-commerce do fornecedor em operações B2B. Com essa funcionalidade, a finalização da compra é realizada externamente, diretamente pelo sistema de eprocurement da empresa.

Reviewer 3: > O fluxo de Punchout permite que compradores integrem seus sistemas de procurement diretamente ao site de e-commerce da loja. O Checkout para Punchout da VTEX é um checkout adaptado para esse tipo de integração, que permite que o usuário retorne o carrinho montado para seu sistema de procurement ("punch back"), e finalize a compra por ali.

"Punchout" se refere à integração e também à ação de "sair" do procurement system para o site.
"Punchback" se refere à integração e também à ação de "voltar" para o procurement system com o carrinho montado. Aqui é mais correto que se diga que ocheckout adaptado permite o punch back para o sistema de procurement

### Endereços de cobrança

> A funcionalidade Endereços de cobrança permite múltiplos endereços de cobrança por organização e possibilitar que os compradores selecionem explicitamente o endereço de cobrança correto durante o checkout. A funcionalidade garante que o endereço de cobrança registrado no pedido reflita com precisão a realidade fiscal da transação e seja propagado de forma consistente para os sistemas posteriores.

Reviewer 3: > A funcionalidade Endereços de cobrança permite que os compradores selecionem explicitamente o endereço de cobrança correto durante o checkout. A funcionalidade garante que o endereço de cobrança registrado no pedido reflita com precisão a realidade fiscal da transação e seja propagado de forma consistente para os sistemas posteriores.

![Endereços de cobrança](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout_3.png)

### Entrega agendada

A [Entrega agendada](/docs/pt/tutorials/configurar-entrega-agendada-fastcheckout) permite às lojas VTEX oferecer aos clientes a possibilidade de escolher o melhor dia e horário para receber suas compras. Essa configuração é realizada diretamente nas políticas de envio e definem os períodos disponíveis para entregas.

### Políticas de compra

> O sistema de [Políticas de compra](https://developers.vtex.com/docs/api-reference/buying-policies-api) suporta regras dinâmicas que os lojistas podem configurar para se adequar aos seus processos específicos de autorização. A funcionalidade disponibiliza um mecanismo para que os usuários registrem expressões de regras customizadas, oferecendo maior flexibilidade e extensibilidade. O processo de aprovação é hierárquico e utiliza um sistema de avaliação baseado em prioridade.

Reviewer 3: > O sistema de [Políticas de compra](https://developers.vtex.com/docs/api-reference/buying-policies-api) suporta regras dinâmicas que os compradores podem configurar para inserir suas regras de compliance dentro do processo de compra. Gestores de organizações compradoras podem definir regras customizadas para que pedidos sejam encaminhados para aprovação antes de serem confirmados, ou mesmo sejam impedidos de serem completados no ato da compra.

### Assembly Options

O recurso de [Assembly Options](https://help.vtex.com/pt/docs/tutorials/assembly-options) permite agrupar itens de um kit em um único elemento na lista do carrinho. Esse elemento exibe informações detalhadas sobre a composição do kit e oferece controle das quantidades de cada item.

![Assembly Options](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout_4.png)

### Cartões salvos

> A funcionalidade de [salvar cartões](https://developers.vtex.com/docs/api-reference/checkout-custom-card-payment-api) permite fluxos de tokenização de cartões de crédito na VTEX, abrangendo gerenciamento de perfis, operações do cofre de cartões, processos de checkout, interações com gateways de pagamento e fluxos de autenticação.

Reviewer 3: > O Buyer Portal Checkout permite a utilização de cartões de créditos compartilhados pela organização. Ele também permite a possibilidade de [salvar cartões](https://developers.vtex.com/docs/api-reference/checkout-custom-card-payment-api) a usuários que possuam essa permissão para que possam ser utilizados em transações futuras. Integrado ao fluxo de tokenização de cartões de crédito da VTEX, as informações são processadas segundo as regras de PCI, fornecendo segurança ao processo.

> ### Gerenciador de Endereços

Reviewer 3: > Endereços de entrega salvos

> O Gerenciador de Endereços permite que usuários com o nível de permissão adequado adicionem e editem endereços durante uma compra B2B. O endereço utilizado para finalizar a compra ficará salvo na lista de endereços para futuras compras.

Reviewer 3: > No Buyer Portal Checkout, usuários podem selecionar endereços de entrega a partir da lista fornecida pela sua organização. Usuários com a permissão correta também podem registrar novos endereços e editar existentes diretamente na experiência de compra do checkout.

![Gerenciador de Endereços](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout_5.png)

> ### Entrega múltipla (Multidelivery)

Reviewer 3: Múltiplas opções de entrega (Multidelivery)

> A funcionalidade permite que os compradores escolham uma [opção de envio](https://help.vtex.com/pt/docs/tutorials/opcoes-de-envio-beta) para cada item, proporcionando mais  flexibilidade e melhoria na experiência do usuário contribuem para a redução do abandono de carrinho e beneficiam lojistas em modelos de negócio B2B.

Reviewer 3: > A funcionalidade permite que os compradores escolham diferentes combinações de envio opções de envio para cada item, proporcionando mais flexibilidade para o fulfillment da compra.

A feature de opções de envio não está necessariaente integrada ao checkout buyer portal

![Multidelivery](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout_6.png)

### Campos contábeis

> [Campos de dados customizaveis](https://help.vtex.com/pt/docs/tutorials/campos-contabeis), configurados pela organização compradora para coletar informações administrativas ou financeiras, como centro de custo, número de pedido interno (PO) e outros dados de controle.

Reviewer 2: Aqui diria que não se limita a campos contáveis. O melhor termo talvez seria "Campos customizáveis da conta"? A ideia é que agora, na jornada B2B, a conta pode adicionar campos específicos de sua jornada e encaixá-los na etapa de Cart (ex: Centro de Custo), Endereço (ex: Location), Payment (PO Number).

Reviewer 3: opa, o nome oficial é "accounting fields" mesmo. A descrição está correta

![Campos contábeis](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout_7.png)

### Pickup Store

O Buyer Portal Checkout permite que os compradores localizem e selecionem uma loja para retirada do pedido durante o processo de finalização de compra. É possível buscar por endereço ou CEP, filtrar por raio e verificar a disponibilidade dos itens em tempo real para cada local.

> ![Campos contábeis](https://cdn.statically.io/gh/vtexdocs/help-center-content/refs/heads/main/docs/pt/tutorials/checkout/buyer-portal-checkout/introducao-buyer-portal-checkout_8.png)

Reviewer 2: Ultima duvida, as demais funcionalidades basicas do Checkout, como adicionar/editar/remover itens, seleção de endereço, seleção de pagamentos e tela de fechamento de compras, deveriam ser listadas aqui?

Reviewer 3: acho interessante que fossem listadas também, ou pelo menso mencionadas no início
